src/odemis/driver/test/simcam_test.py

```diff
@@ -108,12 +118,14 @@ def test_roi(self):
         self.camera.resolution.value = self.camera.resolution.range[1]
         self.camera.translation.value = (0, 0)
 
-#     @unittest.skip("simple")
+    # @unittest.skip("simple")
     def test_acquire(self):
         self.assertGreaterEqual(len(self.camera.shape), 3)
         exposure = 0.1
         self._ensureExp(exposure)
 
+        self.camera.updateMetadata({model.MD_IN_WL: (500e-9, 700e-9)})
```

Comment:
You need to try with different metadata. First try when there is no MD_IN_WL, then with this value, and then with values very far, eg, `(900e-9, 950e-9)`.